base/cvd/cuttlefish/host/commands/cvd/cli/selector/arguments_lexer.cpp

```diff
@@ -79,8 +160,12 @@ ArgumentsLexerBuilder::GenerateFlagPatterns(
   return flag_patterns;
 }
 
-Result<std::unique_ptr<ArgumentsLexer>> ArgumentsLexerBuilder::Build(
-    const LexerFlagsSpecification& known_flags) {
+Result<std::unique_ptr<ArgumentsLexer>> ArgumentsLexerBuilder::Build() {
+  LexerFlagsSpecification known_flags{
+      .known_boolean_flags = {SelectorFlags::kVerbosity},
+      .known_value_flags = {SelectorFlags::kGroupName,
+                            SelectorFlags::kInstanceName},
```

Comment:
Probably not for this PR, but knowledge of what these flags are meant to be is duplicated between here and `ParseCommonSelectorArguments` in `selector_common_parser.cpp`. One way to fix that would be for this parser to return the cvd flags already parsed instead of a list of strings that need to be parsed again.

Reply:
Added a comment in both locations referencing the other place to update.

This is a downside of pushing down hardcoded values. This might be possible to address later in deduplicating `CvdFlag` and `Flag.